website/static/js/apiApplication.js

```diff
@@ -0,0 +1,434 @@
+/*
+ *  Knockout views for the Developer application pages: List and Create/Edit pages.
+ */
+
+
+'use strict';
+
+var $ = require('jquery');
+var bootbox = require('bootbox');
+var historyjs = require('exports?History!history');
+
+var ko = require('knockout');
+require('knockout.validation');
+require('knockout.punches');
```

Comment:
Are you using knockout punches in your templates? Only import it if you think it's necessary or makes the template syntax significantly cleaner; otherwise, remove it.